src/mono/wasm/debugger/BrowserDebugProxy/DebugStore.cs

```diff
@@ -541,10 +541,19 @@ public unsafe AssemblyInfo(string url, byte[] assembly, byte[] pdb)
             asmStream = new MemoryStream(assembly);
             peReader = new PEReader(asmStream);
             asmMetadataReader = PEReaderExtensions.GetMetadataReader(peReader);
+            Name = asmMetadataReader.GetAssemblyDefinition().GetAssemblyName().Name + ".dll";
+            AssemblyNameUnqualified = asmMetadataReader.GetAssemblyDefinition().GetAssemblyName().Name + ".dll";
             if (pdb != null)
             {
                 pdbStream = new MemoryStream(pdb);
```

Comment:
IIUC, `pdbStream` doesn't really ever get used again. We instantiate it here, and use it to get the `pdbMetadataReader`. And in other methods, we re-instantiate.
And same for `asmReader`.
If so, then we can remove the fields, and dispose these.